gtsam/linear/NoiseModel.h

```diff
@@ -776,7 +776,7 @@ namespace gtsam {
 
         Huber(double k = 1.345, const ReweightScheme reweight = Block);
         double weight(double error) const {
-          return (error < k_) ? (1.0) : (k_ / fabs(error));
+          return (std::abs(error) < k_) ? (1.0) : (k_ / fabs(error));
```

Comment:
I propose
```
const double absError = std::abs(error);
return (absError < k_) ? (1.0) : (k_ / absError);
```

Reply:
Updated both.

Reply:
This reminded me again the beauty of functional programming :)